content/docs/command-reference/add.md

```diff
@@ -378,3 +380,49 @@ system that can handle it), they can use `dvc pull` as usual:
 A       data.xml
 1 file added and 1 file fetched
 ```
+
+## Example: Transfer to the cache
+
+When you have a large dataset in an external location, you may want to add it to
+your cache without actually copying it into the workspace first. This might be
+due to cache and workspace are in separate disks, which only the cache can
+handle that size of data. After the data is saved to your cache, we link it to
+your workspace with the
+[preffered links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache).
+
+Let's initalize a DVC project;
+
+```dvc
+$ mkdir example # workspace
+$ cd example
+$ git init
+$ dvc init
+```
+
+Afterwards, let's setup a shared cache by following
+[this](https://dvc.org/doc/use-cases/shared-development-server#preparation)
+tutorial. When it is ready to go, we can add `data.xml` to our cache directly;
+
+```
+$ dvc add https://data.dvc.org/get-started/data.xml -o data.xml
+```
```

Comment:
Yeah for now I think delete the part about shared cache and just leave something like:

````suggestion
Now we can add a `data.xml` file via HTTP for example, putting it a local path
in our project:

```
$ mkdir data
$ dvc add https://data.dvc.org/get-started/data.xml --out data/data.xml
...
```
````

And instead of `...`, maybe show a sample command output here?

Reply:
Also not sure about the https://data.dvc.org/get-started/data.xml example... We could keep it hypothetical with some abstract path like `/mnt/nfs/raw/data.xml` (or some better idea) or ssh://user@example.com/raw/data.xml

Idk, I guess it's not very important but data.dvc.org is kind of an internal company thing (not a secret but still).